### DISCUSSION TITLE
adjusting length of an Array

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#266] [Web] [answered]
Hey everyone,

I have the following user experience issue:

In my countdown, the user can choose the time that should expire. The problem, however, is that the numbers  start from one, so the minimum value you can enter is one. I use a Array with a map method to save time, and so that I dont have to assign like 60 different option tags inside the Select. 
`	const mins = Array.from({length: 59}).map((_, i) => {
		return i + 1;`

<img width="265" alt="Bildschirmfoto 2022-10-14 um 00 06 34" src="https://user-images.githubusercontent.com/109968361/195721825-64236f20-3568-4113-b4d8-ad93402cf4a3.png">

Is there a way to adjust the Array to my needs, so that it starts from 0 ?

**Comment by pixelass** [comment, marked as answer]
Just try to not add `1` to the index. :) 

```js
const mins = Array.from({length: 59}).map((_, index) => { return index };
```

**Reply by DennisWarchocki** [reply]
Well.... Thanks alot 👍
